external/README.md

```diff
@@ -2,6 +2,25 @@
 
 Third Party container tests help verify that the adoptium binaries are *good* by running a variety of Java applications inside of Docker containers. adoptium/aqa-tests/[Issue #172](https://github.com/adoptium/aqa-tests/issues/172) lists the applications that we have initially targeted to best exercise the adoptium binaries.  For each application, we choose to run a selection of their functional tests.
 
+## Podman, docker and others. Sudo, runas and others
+
+The toolchain understands two environment variables:
+```
+EXTERNAL_AQA_RUNNER=docker/podman/...
```

Comment:
Should that be `|` characters instead of `/` to indicate you choose one of these options?  Same below

Reply:
Sure, why not. I do not have strong preference